src/Data/Store/Internal.hs

```diff
@@ -46,6 +46,8 @@ module Data.Store.Internal
     , sizeSet, pokeSet, peekSet
     -- ** Store instances in terms of IsMap
     , sizeMap, pokeMap, peekMap
+    -- ** Store instances in terms of IArray
+    , sizeArray, pokeArray, peekArray
```

Comment:
Actually I don't believe that there are a lot of custom `IArray` instances out there…

Should I delete those exports?